app/src/main/res/values/styles.xml

```diff
@@ -115,6 +115,97 @@
         <item name="preferenceTheme">@style/PreferenceThemeOverlay.v14.Material</item>
     </style>
 
+    <!-- DrawerArrowStyle is used to change the color of hamburger-icon to white -->
+    <style name="DrawerArrowStyle" parent="Widget.AppCompat.DrawerArrowToggle">
+        <item name="color">@android:color/white</item>
+    </style>
+
+    <style name="ToolbarTheme" parent="ThemeOverlay.AppCompat.ActionBar">
```

Comment:
Isn't there a way to inherit all of the icon definitions from the dark theme, so that they don't have to be specified two times?